package.json

```diff
@@ -32,7 +32,7 @@
   "scripts": {
     "start": "webpack-dev-server --inline --hot --content-base dist/",
     "build": "npm-run-all lint:build clean build:webpack",
-    "build:webpack": "webpack --mode production",
+    "build:webpack": "node --max_old_space_size=4096 node_modules/.bin/webpack --mode production",
```

Comment:
@kswenson the build kept crashing with an out of memory error after updating versions - similar to the error I saw after installing `mathjs`:

https://travis-ci.org/concord-consortium/collaborative-learning/builds/505930737

I found a suggestion to increase `max_old_space_size` here and that seemed to solve the problem:

Does that all seem reasonable to you?

Reply:
I'm fine with the `max_old_space_size` setting. I'm slightly leery of the need to hard-code the path to webpack. Out of curiosity, did you try this suggestion from further down in the same thread?
```
"scripts": {
    "build": "cross-env NODE_OPTIONS=--max_old_space_size=4096 webpack"
}
```

Reply:
Looks like this worked and I agree it's much cleaner. PR should now be good to go!